shacl2md.py

```diff
@@ -61,13 +62,28 @@ def get_subclasses(g, c, lang):
 
 
 def get_classes(g, lang):
+    def iri_func(p):
+        return p['iri']
     for c in g.query(GET_CLASSES, initBindings={"lang": Literal(lang)}):
+        ungrouped_properties = list(get_properties(g, c.iri, lang))
+        properties = []
```

Comment:
kan je wat commentaar aan de code toevoegen zodat het duidelijk is wat hier gebeurd?

Reply:
Deze was omdat er in de originele SHACL-file soms meerdere property shapes per property zaten. Hierbij worden deze samengevoegd indien alle waarden die elke property shape definiëren gelijk zijn of bij de ene niet voorkomen.